data/text/default.ts

```diff
@@ -146,7 +146,7 @@ export const DefaultText: {[k: string]: DefaultText} = {
 	},
 	spd: {
 		statName: "Sp. Def",
-		statShortName: "SpD",
+		statShortName: "SpDe",
```

Comment:
The common abbreviation for Sp. Def has always been SpD, so that won't change either.